cdk/apps/dracarys/README.md

```diff
@@ -0,0 +1 @@
+This is a CDK repo deploying a lambda that takes in a presign URL and a prefix, runs Dracarys on the file downloaded from the presign URL, and writes the output to S3.
```

Comment:
I thought it takes a directory as input and figures out by itself which files to look at/ingest?
Perhaps document the input parameters? 
(I know I can pull it from the code, but I am lazy ;-) )

Reply:
And perhaps a link to `Dracarys` for context?